src/ui/components/focus/AutoFocusGuide.tsx

```diff
@@ -12,28 +12,23 @@ export interface AutoFocusGuideProps {
 const FOCUS_GUIDE_STYLE: ViewStyle = {
   width: '100%',
   justifyContent: 'center',
-  alignItems: 'center'
+  alignItems: 'center',
 };
 
-const TVFocusGuideView = lazy((): Promise<{ default: React.ComponentType<any> }> => {
```

Comment:
This `lazy` had the purpose of ensuring we only ever do the `console.warn` *once*. Without it, it could get logged multiple times. See https://github.com/THEOplayer/react-native-theoplayer-ui/pull/49#discussion_r2054491588.

If you don't like that, you can probably also do it with a boolean flag at the module level.